src/Game.cpp

```diff
@@ -259,6 +267,25 @@ bool Game::running() const
 	return isRunning;
 }
 
-bool Game::getWinner() {
+bool Game::getWinner() const {
 	return this->winner;
-}
\ No newline at end of file
+}
+
+void Game::refreshPlayers() {
```

Comment:
too much duplicate code, maybe move to a separate function `resetPlayer(coords)`